Dockerfile

```diff
@@ -1,5 +1,5 @@
 # stage 1 Generate celestia-appd Binary
-FROM docker.io/golang:1.18.10-alpine3.17 as builder
```

Comment:
if this is the version of Go used to compile and make the celestia-app binary, I think it shouldn't be upgraded until we update the Go version in go.mod of this repo